src/test/java/com/amazonaws/services/kinesis/multilang/MultiLangDaemonConfigTest.java

```diff
@@ -88,4 +102,126 @@ public void propertyValidation() {
         }
     }
 
+    @Test public void KinesisClientLibConfiguration_should_get_proxy_info_from_properties_file() {
```

Comment:
Could you keep the annotation before the test method
```java
@Test
public void propertyValidation() {
    // Test goes here
}
```

Could you use the naming standards, use camel case in the method name and not underscores and add test at the start for eg: testProxyInfoFromPropertiesFile.

Reply:
Your formatter is what puts the @Test annotation on the same line as the test...

I'll change the name of the tests

Reply:
I will suggest though that "test" as a prefix doesn't add anything, especially with a @Test annotation.  Also, the other tests in there don't follow this convention, nor do the other test names actually tell you what the test is checking...